### ISSUE TITLE
Need to add instructions for using the setup-project.sh script

### ISSUE THREAD

**Issue by bawilless** [avnet-iotconnect/iotc-sphereos-sdk#2]
I don't see anything in the iotc-spheresos-sdk directory repo telling users that they need to run the setup-project.sh script to pull a buildable demo.  This needs to be easy to see and detailed enough so that new users/programmers don't have to do research to figure out how to run the script.  Dumb it down.  The instructions should cover running the script in liunx and windows since most Azure Sphere developers will be using the Windows platform.

**Comment by alanlhc** [comment]
Resolved by adding the followings:
1) Added iotc-c-lib and cJSON as submodules to the repo.
2) Added "Clone the samples" section in the README.md to tell user to clone the repo with submodules using git command.

**Comment by alanlhc** [comment]
1. Added iotc-c-lib and cJSON as submodules to the repo. *[done]*
2. Added "Clone the samples" section in the README.md to tell user to clone the repo with submodules using git command. *[done]*
